code/datums/jobs.dm

```diff
@@ -61,6 +62,7 @@
 	var/list/slot_rhan = list()
 	var/list/items_in_backpack = list() // stop giving everyone a free airtank gosh
 	var/list/items_in_belt = list() // works the same as above but is for jobs that spawn with a belt that can hold things
+	var/access_string = null // used to quickly grab access via string, i.e. "Chief Engineer"
 	var/list/access = list(access_fuck_all) // Please define in global get_access() proc (access.dm), so it can also be used by bots etc.
```

Comment:
comment should specify that it completely overrides `var/list/access` if non-null